.docker-compose.vital-strategies-theme.yaml

```diff
@@ -52,3 +56,7 @@ services:
                                    CREATE EXTENSION IF NOT EXISTS postgis_topology;         \
                                    CREATE EXTENSION IF NOT EXISTS fuzzystrmatch;            \
                                    CREATE EXTENSION IF NOT EXISTS postgis_tiger_geocoder;"
+  datapusher:
+    container_name: datapusher
```

Comment:
This is already defined in main compose file. As the entire service is. Having its definition copied here is redundant, unless anything changes or adds to it.